pkg/skaffold/build/cache/cache.go

```diff
@@ -48,11 +48,11 @@ type cache struct {
 	artifactCache    ArtifactCache
 	cacheMutex       sync.RWMutex
 	client           docker.LocalDaemon
-	cfg              docker.Config
+	cfg              Config
 	cacheFile        string
 	imagesAreLocal   bool
 	tryImportMissing bool
-	hashForArtifact  func(ctx context.Context, a *latest.Artifact) (string, error)
```

Comment:
Why not just replace this with the `artifactHasher`?

Reply:
I can do this but then `cache` will need a `Reset` method that flushes the `artifactHasher` since I want to recompute the values _on every dev loop_ but don't want to recompute any value _within a dev loop_. I think the current implementation of instantiating `artifactHasher` on every `lookup` is probably better.

Reply:
Done